### PR TITLE
feat: added alert section

### PR DESCRIPTION
**Pull request by aryan-13**
- Added alert section
- fixed sidebar

alert

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.alert-box {
+	height: 4rem;
+	font-size: 1.7rem;
+	margin: 1rem;
+	display: flex;
+	align-items: center;
+	padding: 0.5rem 2.5rem;
```
Comment (lannisterrr) [inline components/alert.css:7]: margin and padding can go to seperate files  for some global spacing styles.

File: css/layout/sidebar.css [hunk 2]
```diff
@@ -15,8 +18,9 @@
 	flex-direction: column;
 	align-items: stretch;
 	font-weight: 500;
-	justify-content: center;
+	/* justify-content: center; */
```
Comment (raghav2110) [inline css/layout/sidebar.css:21]: Better to remove unwanted code then commenting.

File: components/alert.css (hunk 1 above)
Comment (aryan-13) [reply components/alert.css:7]: I have them in the utility file. These are individual paddings for the specific element.

File: css/layout/sidebar.css (hunk 2 above)
Comment (aryan-13) [reply css/layout/sidebar.css:21]: Actually this is for my reference as there's a bug here.

**Pull request merged**